lib/two_player_game.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'player'
+require_relative 'game_logic'
+
+class TwoPlayerGame
```

Comment:
Totally agree, that's a great suggestion. I wasn't too happy with how I did this really, especially because of the similarity in methods between the two classes. I wanted this to be a single class really, but it was much easier to implement this way. I didn't think about using a parent class at all, but that would definitely have been the best way to do this.